app/util/api/jira_clients.py

```diff
@@ -195,3 +198,16 @@ def get_locale(self):
         api_url = f'{self.host}/rest/api/2/myself'
         user_properties = self.get(api_url, "Could not retrieve user")
         return user_properties.json()['locale']
+
+    def get_applications_properties(self):
+        api_url = f'{self.host}/rest/api/2/application-properties'
+        user_properties = self.get(api_url, "Could not retrieve application properties")
+        return user_properties.json()
+
+    def check_rte_status(self):
+        app_prop = self.get_applications_properties()
+        rte = [i['value'] for i in app_prop if i['id'] == 'jira.rte.enabled'][0]
```

Comment:
`rte=app_prop.get('jira.rte.enabled')
return rte=='true'`

Reply:
Unfortunately, we can't use _get()_ method because of next dict scheme
` {'id': 'jira.rte.enabled',
  'key': 'jira.rte.enabled',
  'name': 'jira.rte.enabled',
  'type': 'boolean',
  'value': 'true'},`